lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -135,6 +136,61 @@ def add_airline_data(post, options)
         post['order']['additionalInput']['airlineData'] = airline_data
       end
 
+      def add_lodging_data(post, options)
+        return unless lodging_options = options[:lodging_data]
```

Comment:
Just as an aside comment, I personally prefer this pattern where we have a guard clause inside the method over making the method call conditional as in
```add_lodging_data(post, options) if options[:lodging_data]```

Reply:
Same. I actually refactored the `add_airline_data` to do the same. It just feels a bit cleaner and reduces the nesting as we fill in the body of the request.